contracts/exchangeV3/offchainWrappersV3/KyberAggregatorWrapper.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+import "../../utils/SafeERC20.sol";
+import "../../DS/DSMath.sol";
+import "../../auth/AdminAuth.sol";
+import "../DFSExchangeHelper.sol";
+import "../../interfaces/exchange/IOffchainWrapper.sol";
+import "../../utils/KyberInputScalingHelper.sol";
+
+contract KyberAggregatorWrapper is IOffchainWrapper, DFSExchangeHelper, AdminAuth, DSMath, KyberInputScalingHelper{
+
+    using TokenUtils for address;
+
+    /// @notice offchainData.callData should be this struct encoded
+    struct KyberL1Calldata{
```

Comment:
No need for struct packing, the calldata can be like in other offchain wrappers, just use array slices from to separate the selector and rest of the call.

Change:
`ExchangeData memory _exData ->  ExchangeData calldata _exData`

And just use:
      `  bytes memory scaledCalldata = getScaledInputData(bytes4(_exData.offchainData.callData[:4]), _exData.offchainData.callData[4:], _exData.srcAmount);`